yarn.lock

```diff
@@ -9389,11 +9389,16 @@ typedarray@^0.0.6, typedarray@~0.0.5:
   version "0.0.6"
   resolved "https://registry.yarnpkg.com/typedarray/-/typedarray-0.0.6.tgz#867ac74e3864187b1d3d47d996a78ec5c8830777"
 
-ua-parser-js@0.7.21, ua-parser-js@^0.7.13, ua-parser-js@^0.7.21:
+ua-parser-js@0.7.21, ua-parser-js@^0.7.13:
```

Comment:
why do we still have deps on 0.7.21 and 0.7.13?

Reply:
@OrenMe 0.7.13 removed (was by mistake from playkit dependency)
0.7.21 comes from karma and stream-http packages, both are dev dependencies so not relevant.